openshift-201/resource-mgmt.md

```diff
@@ -114,19 +114,19 @@ spec:
             value: "20"
           command: ["/opt/rh/httpd24/root/usr/bin/ab"]
           args: ["-dSrk", "-c $(CONCURRENCY)", "-n $(REQUESTS)", "https://$(SERVICE_HOST):$(SERVICE_PORT)/index.html"]
-  
+
 ```
-**Note:** As we don't set limits and request specifically in the deployment the default LimitRange will apply.
+**Note:** As we don't set limits and request specifically in the deployment the default LimitRange will apply. Run `oc describe LimitRange/default-limits` to see what is set as defaults.
 
-From the web console if you change to developer view and navigate to the Monitoring tab select your nginx deployment. You should see the load-test pod traffic increasing metrics for our pod.
+From the web console if you change to developer view and navigate to the Observe tab select your nginx deployment. You should see the load-test pod traffic increasing CPU and Memory usage metrics for nginx workload.
```

Comment:
Just noting that the change of the name of the tab (from Monitoring -> Observe) will potentially impact other areas of these lab instructions, screenshots, and the 101 lab.

Reply:
working my way through!

Reply:
I've gone in to the 101 lab and updated all text references to the 'monitoring' button and changed it there too.